src/services/AttributeStore.ts

```diff
@@ -0,0 +1,12 @@
+export interface AttributeStore {
```

Comment:
Do you see “attribute” here as a feature of any given entity in the starter? If so I would imagine `updateGatewayName()`, `updateGatewaySomethingElse()`, etc would fit here too.

Reply:
Yes that would be good. Or we can split into two interfaces for attributes on Gateways and Nodes, with attribute store extending both of those.